projects/golang/go/1.21/rpmbuild/SPECS/golang.spec

```diff
@@ -527,7 +527,10 @@ fi
 %endif
 
 %changelog
-* Fri May 10 2024 Sajia Zafreen <szafreen@amazon.com> - 1.21.10-10
+* Fri Jun 05 2024 Saurabh Parekh <sjparekh@amazon.com> - 1.21.11-12
```

Comment:
This is causing a build warning: `warning: bogus date in %changelog: Fri Jun 05 2024 Saurabh Parekh <sjparekh@amazon.com> - 1.21.11-12` Can we fix this to be `Wed Jun 05 2024...`

Reply:
Fixed it in a new PR